### PR TITLE
[DOCS-7497] Add Complex Forms feature to Mobile Workspace 1.9

### PR DESCRIPTION
**Pull request by mbujak-hyland**

### CODE REVIEW

**Comment by mbujak-hyland** [comment]
Amanpal Singh reviewed the draft and uploaded it to the Slack channel (#eng-device-support-team)

The content has been modified per (Amanpal's) technical suggestions and is ready for review.

File: mobile-workspace/latest/using/index.md [hunk 1]
```diff
@@ -270,6 +270,264 @@ To set up a workflow, you can either:
...
+* Display value 
+* Display text 
+
+### Workflow with Task
+
+The workflow process manages the completion of all tasks in a specific sequence. You can view and filter workflow processes, start new ones, or track ongoing processes. The workflow process is complete when all associated tasks are finished.
+
+> **Note:** A workflow with a single task is completed once this task is finished.
+
+To initiate a workflow process and start completing tasks, see [Start a Workflow](####start-a-workflow).
```
Comment (anxumalo) [inline mobile-workspace/latest/using/index.md:308]: **For all in-page anchors:** You only need a single `#` character: 

```suggestion
To initiate a workflow process and start completing tasks, see [Start a Workflow](#start-a-workflow).

```

File: mobile-workspace/latest/using/index.md [hunk 2]
```diff
@@ -270,6 +270,264 @@ To set up a workflow, you can either:
...
+
+5. Tap **Start workflow**. 
+
+    The workflow has been started, and it is listed in the Workflows tab under **Active**. 
+
+Once a workflow has been started, you can proceed to complete all tasks within it. To complete
+tasks, see [Complete Task Forms](####complete-task-forms).
+
+> **Note:** If a workflow process assigns tasks to groups (referred to as a pooled process), you must first
+claim the task before you can start working on it. To claim a task, see [Claim a Task](####claim-a-task).
```
Comment (anxumalo) [inline mobile-workspace/latest/using/index.md:342]: Suggested fix for note & link:
```suggestion
> **Note:** If a workflow process assigns tasks to groups (referred to as a pooled process), you must first claim the task before you can start working on it. For more information, see [Claim a Task](#claim-a-task).
```

File: mobile-workspace/latest/using/index.md [hunk 3]
```diff
@@ -270,6 +270,264 @@ To set up a workflow, you can either:
...
+1. From the **Workflows** tab, open a workflow.
+    
+    The details of the workflow are displayed.
+
+2. Tap **Tasks**.
+
+    The Tasks screen is displayed with a task to complete at the top.
+
+3. Open the task and fill out the form. Available fields and components may vary depending on the
+task.
```
Comment (anxumalo) [inline mobile-workspace/latest/using/index.md:431]:
```suggestion
3. Open the task and fill out the form. Available fields and components may vary depending on the task.
```

File: mobile-workspace/latest/using/index.md [hunk 4]
```diff
@@ -270,6 +270,264 @@ To set up a workflow, you can either:
...
+
+> **Note:** If a workflow process assigns tasks to groups (referred to as a pooled process), you must first
+claim the task before you can start working on it. To claim a task, see [Claim a Task](####claim-a-task).
+
+To view and filter workflows, see [View and Filter Workflows](####view-and-filter-workflows).
+
+#### View and Filter Workflows
+
+All workflows created by you or assigned to you are available from the **Tasks** menu. You can view workflow's
+details after selecting a workflow from the **Workflows** tab.
```
Comment (anxumalo) [inline mobile-workspace/latest/using/index.md:349]:
```suggestion
All workflows created by you or assigned to you are available from the **Tasks** menu. You can view the workflow details after selecting a workflow from the **Workflows** tab.
```

**Review by anxumalo — APPROVED** [review]

**Pull request merged**